nimble/host/audio/services/bass/src/ble_audio_svc_bass.c

```diff
@@ -489,7 +490,18 @@ ble_svc_audio_bass_modify_source(uint8_t *data, uint16_t data_len, uint16_t conn
 
     for (i = 0; i < operation.modify_source.num_subgroups; i++) {
         operation.modify_source.bis_sync[i] = get_le32(&data[offset]);
+        rcv_state->state.subgroups[i].bis_sync_state = operation.modify_source.bis_sync[i];
```

Comment:
this state will be updated when actual SYNC will happen. At this stage we cannot do that as we overwrite a state.

Reply:
Removed